pkg/kv/kvserver/replica_range_lease.go

```diff
@@ -445,6 +445,8 @@ func (p *pendingLeaseRequest) requestLeaseAsync(
 	return nil
 }
 
+var logFailedHeartbeatOwnLiveness = log.Every(10 * time.Second)
```

Comment:
nit: consider 6 seconds, i.e. the lease interval.

Reply:
Not sure why that's better - I'd rather avoid the illusion that this number is somehow special.